src/main/java/build/buildfarm/worker/Executor.java

```diff
@@ -101,7 +104,7 @@ private long runInterruptible(Stopwatch stopwatch, ResourceLimits limits)
       metadata = operationContext.operation.getMetadata().unpack(ExecuteOperationMetadata.class);
     } catch (InvalidProtocolBufferException e) {
       log.log(Level.SEVERE, "invalid execute operation metadata", e);
-      return 0;
+      return ZERO_MICROSECONDS;
```

Comment:
This was rather annoying as Checker Framework refused to return "unit-less" 0 constant.

(arguably, 0 probably doesn't need units as 0 seconds == 0 minutes, but I suppose 0 degrees Celsius is _not_ the same as 0 degrees Fahrenheit)